src/test/java/org/opentripplanner/graph_builder/module/osm/WalkableAreaBuilderTest.java

```diff
@@ -70,14 +70,17 @@ public void testSetup(final TestInfo testInfo) {
       : walkableAreaBuilder::buildWithoutVisibility;
 
     areaGroups.forEach(build);
+    return graph;
   }
 
   // -- Tests --
 
   @Test
   @OsmFile("lund-station-sweden.osm.pbf")
   @Visibility(true)
-  public void test_calculate_vertices_area() {
+  @MaxAreaNodes(5)
+  public void test_calculate_vertices_area(TestInfo testInfo) {
```

Comment:
Is using underscore in test method named ok in our conventions?

Reply:
It's not. I don't know how that managed to get past review. I'm in favour of changing it.